packages/tools/client-debugger/client-debugger-view/tsconfig.json

```diff
@@ -9,5 +9,6 @@
 		"types": ["jest"],
 		"skipLibCheck": true,
 	},
+	"exclude": ["node_modules/@fluentui/react-components/**/*"],
```

Comment:
This ended up not being necessary, tight? `skipLibCheck` was all we needed?

Reply:
I think so, yes. I felt skipping the check for every lib might not be a good idea and to only apply the skip to the react-components library which threw the build error. I'm good to remove this @rohandubal

Reply:
Even in that case, I think this doesn't produce the intended effect. If the base tsconfig.json file already excludes all of `node_modules`, I'm not sure we can do anything about that; and if I recall correctly, overriding `exclude` in "child" tsonfig.json files just adds to the exclusions from parent ones, it doesn't replace them.